bHY6yh-d30103/cod-bqe7s7_bHY6yh-d30103.xml

```diff
@@ -0,0 +1,491 @@
+               <lb ed="#P"/><ref xml:id="bHY6yh-d30103-Rd1e920">Ibidem nono capitulo</ref> <quote xml:id="bHY6yh-d30103-Qd1e823">non est amicitia in questu
+               <lb ed="#P"/>non in assentatione sed in veritate et fide.
+               <pb ed="#P" n="60-v"/>
+               <cb ed="#P" n="a"/>
+               <lb ed="#P"/>Est autem amicitia suauissimus comes sin<lb ed="#P" break="no"/>ceritas 
+               affectationum mutua obsequela rerum ho<lb ed="#P" break="no"/>nestarum 
+               crebra collatio parilitas volunta<lb ed="#P" break="no"/>tum
+               iugis et inuiolabilis fides.</quote>
+            </p>
+            <p xml:id="bHY6yh-d30103-d1e838">
+               <lb ed="#P"/><ref xml:id="bHY6yh-d30103-Rd1e939">Ibidem xv capitulo</ref> <quote xml:id="bHY6yh-d30103-Qd1e840">linguosos ad amicitie secre<lb ed="#P" break="no"/>ta 
+               non arbitror eligendos fides in amici<lb ed="#P" break="no"/>tia 
+               maximum locum tenet.</quote>
+            </p>
+            <p xml:id="bHY6yh-d30103-d1e848">
+               <lb ed="#P"/><ref xml:id="bHY6yh-d30103-Rd1e953">Ibidem xvii capitulo</ref> <quote xml:id="bHY6yh-d30103-Qd1e852">Solidiores inueniuntur plerumque 
+               <lb ed="#P"/>pauperum amicitie quam magnatum. Tanto 
+               <lb ed="#P"/>purior est erga parentem caritatis affectio
+               <lb ed="#P"/>quo nulla circa eum sub est adulationis 
+               <lb ed="#P"/>aut emulationis occasio.</quote>
+            </p>
+            <p xml:id="bHY6yh-d30103-d1e864">
+               <lb ed="#P"/><ref xml:id="bHY6yh-d30103-Rd1e969">Ibidem xxii capitulo</ref> <quote xml:id="bHY6yh-d30103-Qd1e866">Quicquid amico impende<lb ed="#P" break="no"/>ris 
+               nunquam improperes.</quote>
+            </p>
+            <p xml:id="bHY6yh-d30103-d1e872">
+               <lb ed="#P"/><ref xml:id="bHY6yh-d30103-Rd1e979">Ibidem xxxiiii capitulo</ref> <quote xml:id="bHY6yh-d30103-Qd1e874">omnis assentatio sit procul 
+               <lb ed="#P"/>a conditione amicitie aliena. Simula<lb ed="#P" break="no"/>tio 
+               siquidem et assentatio due virulentissime 
+               <lb ed="#P"/>pestes sunt / qui procul ab amicitia relegentur.</quote>
+            </p>
+            <p xml:id="bHY6yh-d30103-d1e886">
+               <lb ed="#P"/><ref xml:id="bHY6yh-d30103-Rd1e995">Ibidem xxv capitulo</ref> <quote xml:id="bHY6yh-d30103-Qd1e888">in honestum est bellum cum
+               <lb ed="#P"/>eo gerere: quem tu familiaritatis secre<lb ed="#P" break="no"/>tarium
+               habuisti soluta amicitia semper 
+               <lb ed="#P"/>ei quem amicum habueris honor est 
+               <lb ed="#P"/>exhibendus.</quote>
+            </p>
+            <p xml:id="bHY6yh-d30103-d1e898">
+               <lb ed="#P"/><ref xml:id="bHY6yh-d30103-Rd1e1009">Ibidem xxvoo capitulo et fi:</ref> <quote xml:id="bHY6yh-d30103-Qd1e900">dilectionis est con<lb ed="#P" break="no"/>sulere 
```

Comment:
I would say the same thing about the colon here. The colon should go outside the <ref> tag as it is not part of the reference, but punctuation separating the ref and the quote. 

```suggestion
               <lb ed="#P"/><ref xml:id="bHY6yh-d30103-Rd1e1009">Ibidem xxvoo capitulo et fi</ref>: <quote xml:id="bHY6yh-d30103-Qd1e900">dilectionis est con<lb ed="#P" break="no"/>sulere 
```